packages/commcare/src/Adaptor.js

```diff
@@ -347,6 +348,119 @@ export function request(method, path, body, options = {}) {
     }
   };
 }
+
+/**
+ * Bulk upload data to CommCare. Accepts an array of objects, converts them into
```

Comment:
Currently the bulk upload works for `case-data` and `lookup-table` not all commcare bulk upload API, Maybe we should say `Bulk upload case-data or lookup-table to CommCare`